test/unit/test-ssr-template-helper.js

```diff
@@ -158,14 +159,14 @@ describes.fakeWin(
 
       describe('applySsrOrCsrTemplate', () => {
         it('should set html template', () => {
-          ssrTemplateHelper.applySsrOrCsrTemplate(
-            {},
-            {html: '<div>some template</div>'}
-          );
-          expect(findAndSetHtmlForTemplate).to.have.been.calledWith(
-            {},
-            '<div>some template</div>'
-          );
+          return ssrTemplateHelper
+            .applySsrOrCsrTemplate({}, {html: '<div>some template</div>'})
+            .then(() => {
+              expect(findAndSetHtmlForTemplate).to.have.been.calledWith(
```

Comment:
Should we test that it's called _and_ returns?

Reply:
Good idea considering the mistake I made above

Reply:
Done